eng/Build.ps1

```diff
@@ -362,14 +362,14 @@ function VerifyAssemblyVersionsAndSymbols() {
     }
 }
 
-function TestUsingMSBuild([string] $testProject, [string] $targetFramework, [string]$testadapterpath, [boolean] $asBackgroundJob = $false) {
+function TestUsingMSBuild([string] $testProject, [string] $targetFramework, [string]$testadapterpath, [boolean] $asBackgroundJob = $false, [string] $settings = "") {
     $dotnetPath = InitializeDotNetCli
     $dotnetExe = Join-Path $dotnetPath "dotnet.exe"
     $projectName = [System.IO.Path]::GetFileNameWithoutExtension($testProject)
-    $testLogPath = "$ArtifactsDir\TestResults\$configuration\${projectName}_$targetFramework.xml"
+    $testLogPath = "$ArtifactsDir\TestResults\$configuration\{assembly}_{framework}.xml"
```

Comment:
I think this will lead to a broken path now, should be reverted.

Reply:
They are expanding tokens specific to XunitXml.TestLogger. See https://github.com/spekt/testlogger/wiki/Logger-Configuration#logfilepath
I should put a comment there.

Reply:
The reason to put it there was to deconflict logs when running dotnet test on a full solution.

Reply:
Oh alright then, thanks!